docs/references/javascript/organization/domains.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Organization Domains
+description: These are all methods on the Organization class that allow you to manage the domains of an organization.
+---
+
+import { Tables } from "@/components/Table";
```

Comment:
We don't need to import components in MDX files, because our marketing repo is rendering our docs repo, and the components exist in the marketing repo.```suggestion
```